unittest/unit/find-missing.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const assert = require('chai').assert;
+
+const proxyquire = require('proxyquire');
+
+const tests = {
+  'api.AbortController': {
+    tests: [{code: '"AbortController" in self', prefix: ''}],
```

Comment:
find-missing.js uses `Object.key(tests)` so there's no need to maintain the detailed structure in mock data. Instead, just `'api.AbortController': {}` or `'api.AbortController': 'dummy'` would do. Something more like the dummy bcd data.